### PR TITLE
BUG fix compilers

### PR DESCRIPTION
**Pull request by beckermr**

### CODE REVIEW

**Comment by beckermr** [comment]
cc @damonge @rmjarvis @c-d-leonard This one should pass and is ready for review!

File: .travis/install.sh [hunk 1]
```diff
@@ -39,12 +39,16 @@ py36)
...
     compilers pkg-config setuptools_scm pytest pandas pytest-cov \
-    cython "camb>=1" isitgr traitlets
+    cython "camb>=1" isitgr traitlets fast-pt
   ;;
 esac;
 
 # we have to activate the cond env before we install this
 source activate test-environment
-pip install https://github.com/JoeMcEwen/FAST-PT/archive/v3.0.2.tar.gz --no-deps
+# pip install https://github.com/JoeMcEwen/FAST-PT/archive/v3.0.2.tar.gz --no-deps
```
Comment (damonge) [inline .travis/install.sh:48]: If line 42 works, can we remove this?

Comment (beckermr) [reply .travis/install.sh:48]: sure

Comment (beckermr) [inline .travis/install.sh:48]:
```suggestion
```

**Comment by beckermr** [comment]
`fast-pt` is not on conda for python 2.7, so I have removed those builds now

**Comment by damonge** [comment]
Yeah, that's fine. About time!

**Comment by damonge** [comment]
Not for this PR, but are we saying explicitly anywhere that we don't support python2 anymore?

**Comment by beckermr** [comment]
It is deprecated and I believe we already did the same.

**Comment by beckermr** [comment]
Also, I lost my permissions to restart and cancel jobs on travis. :/

**Comment by rmjarvis** [comment]
@beckermr Try logging out and back in on Travis.  It gets confused sometimes about your collaborations.

**Comment by beckermr** [comment]
We good to merge @damonge?

**Pull request merged**